x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/import_rules_route.ts

```diff
@@ -166,7 +164,7 @@ export const createImportRulesRoute = (server: ServerFacade): Hapi.ServerRoute =
                       alertsClient,
                       actionsClient,
                       description,
-                      enabled: false,
+                      enabled,
```

Comment:
We are allowing rules to be enabled on import now? Just an observation.

Reply:
Yes, I don't know when this bug was introduced.

If you think about it, it would be a bug and not a feature because when I decide to do an import with "overwrite" set to true and then see all of my current running rules suddenly being disabled I would immediately look at my exported file and notice that the ones I wanted to be turned on are actually turned on in the file I am trying to import. I would begin to wonder why the import feature is turning off my rules when I put into the import file that I wanted to have some of them enabled when I do an import.

Perhaps I am writing automation scripts to import rules between spaces using the API (for example) and need the import to do as it is told exactly. Maybe I am changing the rules before importing through python scripts, etc... At that point I need the import to do exactly as it is told to do.

If we want the user to think about which rules they want to be enabled or disabled or to warn them (at some point) that they have exported rules that are enabled it would be on the export feature that we would warn them and ask them if they want to disable all of the rules before exporting them and add a query parameter to the export.

Then their ndjson file would be a "source of truth" on import and the import would work ok and if they're versioning their saved ndjson files they have confidence they work as they are see in their github repo's.

If they come back to us with this change and tell us they were expecting that imported rules that have `enabled: true` to actually all be `enabled: false` on forums, the fix is for them to just open their imports and change them all to be false or to disable the rules before exporting which I think is the correct path forward.